Example/NativeBase-KitchenSink/src/screens/toast/toast-button.js

```diff
@@ -13,45 +13,43 @@ import {
   Toast
 } from "native-base";
 import styles from "./styles";
+import { TLTRN } from "react-native-acoustic-ea-tealeaf";
 
-class ToastButton extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      showToast: false
-    };
-  }
-  render() {
-    return (
-      <Container style={styles.container}>
-        <Header>
-          <Left>
-            <Button transparent onPress={() => this.props.navigation.goBack()}>
-              <Icon name="arrow-back" />
-            </Button>
-          </Left>
-          <Body>
-            <Title>Toast Button Style</Title>
-          </Body>
-          <Right />
-        </Header>
-
-        <Content padder>
-          <Button
-            onPress={() =>
-              Toast.show({
-                text: "Wrong password!",
-                buttonText: "Okay",
-                buttonTextStyle: { color: "#008000" },
-                buttonStyle: { backgroundColor: "#5cb85c" }
-              })}
-          >
-            <Text>Toast</Text>
+const ToastButton = () => {
+  
+  return (
+    <Container style={styles.container}>
+      <Header>
+        <Left>
+          <Button transparent onPress={() => this.props.navigation.goBack()}>
+            <Icon name="arrow-back" />
           </Button>
-        </Content>
-      </Container>
-    );
-  }
+        </Left>
+        <Body>
+          <Title>Toast Button Style</Title>
+        </Body>
+        <Right />
+      </Header>
+
+      <Content padder>
+        <Button
+          onPress={() => {
+            Toast.show({
+              text: "Wrong password!",
+              buttonText: "Okay",
+              buttonTextStyle: { color: "#008000" },
+              buttonStyle: { backgroundColor: "#5cb85c" },
+              onClose: () => {
```

Comment:
Is there a way to capture automatically? So customer does not need to add one line in onClose?